templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -51,6 +51,39 @@
 						</span>
 					</a>
 				{{end}}
+				{{if .TeamReviewers}}
+					{{range .TeamReviewers}}
+						{{$ReviewerID := .ID}}
+						{{$checked := false}}
+						{{$canChoose := false}}
+						{{$notReviewed := true}}
+
+						{{range $.PullReviewers}}
+							{{if eq .ReviewerID $ReviewerID }}
+								{{$notReviewed = false }}
+								{{if  eq .Type 4 }}
+									{{$checked = true}}
+									{{if $.Permission.IsAdmin}}
+										{{$canChoose = true}}
+									{{end}}
+								{{else}}
+									{{$canChoose = true}}
+								{{end}}
+							{{end}}
+						{{end}}
+
+						{{ if $notReviewed}}
+							{{$canChoose = true}}
+						{{end}}
```

Comment:
Can this logic be somehow be moved out of template? Like adding adding functions to `PullReviewer` or `TeamReviewer` struct?